scheduler/pmc-processor-task/src/test/kotlin/ac/uk/ebi/pmc/importer.data/MongoDocServiceTest.kt

```diff
@@ -36,7 +36,7 @@ internal class MongoDocServiceTest(
     @Test
     fun `save submission with file`(): Unit = runBlocking {
         coEvery { dataRepository.saveSubFile(exampleFile, submission.accNo) } returns fileId
-        coEvery { dataRepository.save(ofType(SubmissionDoc::class)) } returns Unit
+        coEvery { dataRepository.save(ofType(SubmissionDoc::class)) } returns null
```

Comment:
why null now instead of void?

Reply:
Because I changed the save method to be a one-liner so I have to use the Mongok `save` method returning type which is `Void?`.

Since it's a nullable type, I just return null for this tests case.